test/hex_tarball_SUITE.erl

```diff
@@ -126,7 +171,10 @@ build_tools_test(_Config) ->
 
     {ok, #{tarball := Tarball3}} = hex_tarball:create(Metadata, Contents),
     {ok, #{metadata := Metadata2}} = hex_tarball:unpack(Tarball3, memory),
-    false = maps:is_key(<<"build_tools">>, Metadata2),
+
+    %% This check isn't valid anymore since buildtools is required metadata -- Bryan
```

Comment:
See comment

Reply:
Sorry, I am not following, shouldn't both be valid?

```
true = maps:is_key(<<"build_tools">>, Metadata2),
[] = maps:get(<<"build_tools">>, Metadata2),
```

Or are you saying the `build_tools` needs to be a non-empty list? If so please update the test :)

Reply:
WIl re-visit 👍

Reply:
Ahh I see what I was noting here. The test was 

```
 false = maps:is_key(<<"build_tools">>, Metadata2)
```

The test had to change to `true = ...`  and `[] = ...` because build_tools is a required key now. Even if we can't guess it, the key has to be present or validation will fail.